pkg/apis/kops/instancegroup.go

```diff
@@ -109,6 +109,16 @@ type InstanceGroupSpec struct {
 	Kubelet *KubeletConfigSpec `json:"kubelet,omitempty"`
 	// Taints indicates the kubernetes taints for nodes in this group
 	Taints []string `json:"taints,omitempty"`
+	// IAMProfileSpec defines the identity of the cloud group iam profile.
+	IAM *IAMProfileSpec `json:"iam,omitempty"`
+}
+
+// IAMProfileSpec defined the identity of the cloud group iam profile.
+// In EC2 the cloud group is defined by the launch configuration.
+type IAMProfileSpec struct {
+	// Profile of the cloud group iam profile. In aws this is the arn
+	// for the iam instance profile
+	Profile *string `json:"profile,omitempty"`
```

Comment:
How about `Profile is the AWS IAM Profile to attach to instances in this instance group.  Specify the ARN for the IAM instance profile.  (AWS only)`